lib/guest_installation_and_configuration_base.pm

```diff
@@ -333,16 +193,19 @@ sub revise_guest_version_and_build {
         else {
             $self->{guest_build} = 'gm';
         }
-
-        # Replace all guest config values which refer to guest_build via ##guest_build##
-        map { $self->{$_} =~ s/##guest_build##/$self->{guest_build}/g } keys %guest_params;
```

Comment:
why delete it when it can provide some convenience in guest configuration?

Reply:
Guest build already included in guest_installation_media which matches the original design. There is no need to keep it which was introduced along with another module on top of unified_guest_installation.pm. It is not needed and not necessary.

Reply:
Similar with https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18488#discussion_r1501962567

Reply:
Similar with [this comment](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18488#discussion_r1501968246)